com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -34,6 +51,15 @@ public abstract class CameraLabeler
         /// </summary>
         protected virtual void Setup() { }
         /// <summary>
+        /// Called immediately after <see cref="setup"/>. Implement this to initialize labeler's visualization
+        /// capability if one exists <see cref="supportVisualization"/>.
+        /// </summary>
+        protected virtual void SetupVisualizationPanel(GameObject panel) { }
+        /// <summary>
+        /// Called when the labeler's visualization capability is turned on or off.
+        /// </summary>
+        protected virtual void OnVisualizerEnabled(bool enabled) {}
```

Comment:
The name implies (to me) that the visualizer is being enabled, but actually it is either being enabled or disabled. Consider changing to `OnVisualizer<something>Changed`

Reply:
Changed to `OnVisualizerActiveStateChanged()` <- It's kind of lame, but 🤷‍♂️